javascript/nitro/tsconfig.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "./.nitro/types/tsconfig.json"
```

Comment:
where is this file?

Reply:
Hey, this file is generated during `npm install` as postinstall script